pladder/misc.py

```diff
@@ -41,6 +42,20 @@ def capify(text):
     return text.title()
 
 
+def suspektify(text):
+    import random
+    words = text.split(" ")
```

Comment:
```suggestion
```

Reply:
I'm not convinced that the approach with varargs=False is actually preferable. In that case I would have to cast the tuple to a list to be able to modify it in-place, and by then the intention of the function would probably may be clearer just keeping the text.split(" "), giving the function a str->str signature.